Tasks/KubernetesManifestV0/src/models/TaskInputParameters.ts

```diff
@@ -15,6 +16,7 @@ export const secretArguments: string = tl.getInput('secretArguments', false) ||
 export const secretType: string = tl.getInput('secretType', false);
 export const secretName: string = tl.getInput('secretName', false);
 export const dockerRegistryEndpoint: string = tl.getInput('dockerRegistryEndpoint', false);
+export const timeout: string = tl.getInput('timeout', false);
```

Comment:
It's retrieved as a string input. Check if type cast to integer succeeds. If false, you can throw an error here that the value needs be an integer